config.md

```diff
@@ -183,6 +183,10 @@ Note: Any OPTIONAL field MAY also be set to null, which is equivalent to being a
 
     The field contains the system call signal that will be sent to the container to exit. The signal can be a signal name in the format `SIGNAME`, for instance `SIGKILL` or `SIGRTMIN+3`.
 
+  - **ArgsEscaped** *boolean*, OPTIONAL
+
+    `[Deprecated]` - This field is present only for legacy compatibility with Docker and should not be used by new image builders. It is used by Docker for Windows images to indicate that the `Entrypoint` or `Cmd` or both, contains only a single element array, that is a pre-escaped, and combined into a single string `CommandLine`. If `true` the value in `Entrypoint` or `Cmd` should be used as-is to avoid double escaping.
```

Comment:
Could you mention this deprecation in the compatibility matrix too?
https://github.com/opencontainers/image-spec/blob/main/media-types.md#compatibility-matrix

Reply:
Would something like this be sufficient (added the "Moby" bullet points to [here](https://github.com/opencontainers/image-spec/blob/main/media-types.md#applicationvndociimageconfigv1json)):

> ### application/vnd.oci.image.config.v1+json
> 
> **Similar/related schema**
> 
> - [application/vnd.docker.container.image.v1+json](https://github.com/moby/moby/blob/v20.10.8/image/spec/v1.2.md#image-json-description) (Docker Image Spec v1.2)
>   - `.config.Memory`: only present in Docker, and reserved in OCI
>   - `.config.MemorySwap`: only present in Docker, and reserved in OCI
>   - `.config.CpuShares`: only present in Docker, and reserved in OCI
>   - `.config.Healthcheck`: only present in Docker, and reserved in OCI
> - [Moby](https://github.com/moby/moby)
>    -  [`.config.ArgsEscaped`](https://github.com/moby/moby/blob/1a0c15abbb69105be7dab74335f7c9f6bb6f2efa/api/types/container/config.go#L75): Windows-specific Moby extension, deprecated in OCI
> 
> `.config.StopSignal` and `.config.Labels` are accidentally undocumented in Docker Image Spec v1.2, but these fields are not OCI-specific concepts.

Reply:
Done. Sorry for the turn time here I totally forgot about this ask.